<?xml version="1.0"?>
<launch>
  <node name="find_transform" pkg="calibration_rgbdcam_mocap" type="find_transform_node" cwd="node">
    <param name="run_getChessboardPose_inCamCS" value="true"/>
    <param name="rgb_dir" value="$(find calibration_rgbdcam_mocap)/data/input/1_rgb.png"/>
    <param name="depth_dir" value="$(find calibration_rgbdcam_mocap)/data/input/1_depth.png"/>
    <param name="saved_transform_dir" value="$(find calibration_rgbdcam_mocap)/data/output/transformation.txt"/>
    <param name="saved_draw_dir" value="$(find calibration_rgbdcam_mocap)/data/output/1_draw_chessboard.png"/>
    <param name="cx" value="315.5"/>
    <param name="cy" value="239.5"/>
    <param name="fx" value="520.0"/>
    <param name="fy" value="580.0"/>

    <param name="run_getT_camToMarkers" value="true"/>
    <param name="results_dir" value="$(find calibration_rgbdcam_mocap)/data/output/"/>

  </node>
</launch>